examples/audio/eeg_signal.py

```diff
@@ -0,0 +1,527 @@
+"""
+Title: EEG Signal Classification for Stimuli identification
+Author: [Suvaditya Mukherjee](https://twitter.com/halcyonrayes)
+Date created: 2022/11/03
+Last modified: 2022/11/04
+Description: Training a Convolutional model to classify EEG signals produced by exposure to certain stimuli.
+"""
+"""
+## Introduction
+
+The following example explores how we can make a Convolution-based Neural Network to
+perform classification on EEG signals captured when subjects were exposed to different
```

Comment:
can you expand the EEG acronym so those uninitiated can read more?  Please also provide a link

Reply:
Will do that. Also, link to what? The dataset is linked further down below

Reply:
Should I add a Wikipedia link to what EEGs are?